converters/fromZigbee.js

```diff
@@ -7929,6 +7929,9 @@ const converters = {
             case tuya.dataPoints.trsIlluminanceLux:
                 result = {illuminance_lux: value};
                 break;
+            case tuya.dataPoints.trsDetectionData:
+                result = {detection_data: value};
```

Comment:
We should also add this the the `expose` array of the definition.

Reply:
Added the expose to device/tuya.js - at the moment, I'm not sure, what this dp send. Several times, I get an warning: DP104 is unknown, that's the reason for integrating.

Thanks.

Reply:
If we don't know what it means, it would be better to just ignore it.

Reply:
That's why I didn't made  an entry in exposes first.
TuYa Bridge sniffing says, it's called motion detection, but it isn't clear, what that means.
I only added it to catch the warning from z2m "unknown dp".
So, if it's okay for you, I will delete the expose, but will let the DP included in "fromzigbe.js".

Reply:
Ah, okay, thanks